src/sentry/api/serializers/models/organization.py

```diff
@@ -260,6 +260,8 @@ def get_feature_set(
                 extra={
                     "org_features": org_features,
                     "user.id": user.id if not user.is_anonymous else None,
+                    "manager": features.default_manager,
```

Comment:
since we're throwing the kitchen sink for debugging, why not a good ol `id(features.default_manager)` in?

Reply:
Or I guess the default __str__ has a memory identifier?

Reply:
Yeah, default str()/repr() includes a hex address/id of the object.